src/reference/asciidoc/aggregator.adoc

```diff
@@ -114,6 +114,19 @@ This method will be invoked for aggregating messages as follows:
 
 NOTE: In the interest of code simplicity, and promoting best practices such as low coupling, testability, etc., the preferred way of implementing the aggregation logic is through a POJO, and using the XML or annotation support for configuring it in the application.
 
+If the `MessageGroupProcessor` 's `process` method returns a collection, it must be a collection of `Messge<?>` s.
```

Comment:
Oh! Sorry. I see now. Thanks, So never mind.
Only the single `Object` or `Collection<Message<?>>`.